.github/workflows/publish.yml

```diff
@@ -1,50 +0,0 @@
-name: CD
-
-on:
-  workflow_dispatch:
-  release:
-    types: [published]
-
-jobs:
-  publish:
-    name: Publish npm package
-    runs-on: ubuntu-latest
-
-    steps:
-      - uses: actions/checkout@v2
-
-      - name: Use Node 12.X
-        uses: actions/setup-node@v2.1.5
-        with:
-          node-version: 12.x
-          registry-url: https://registry.npmjs.org/
-
-      - run: yarn install
-      - run: yarn build
-
-      - run: npm publish --access public
-        working-directory: './lib'
-        env:
-          NODE_AUTH_TOKEN: ${{ secrets.NPMJS_PUBLISH_ITWIN }}
-
-  fix-title:
-    name: Edit release title
-    runs-on: ubuntu-latest
-    steps:
-      - name: Update release title to match tag name
-        uses: actions/github-script@v4
-        with:
-          script: |
-            const { data: release } = await github.repos.getReleaseByTag({
-              owner: context.repo.owner,
-              repo: context.repo.repo,
-              tag: ${{ github.ref }}
```

Comment:
Why changed this part?

Reply:
previously this was meant to be triggered by a release, whose GITHUB_REF is the tag

now it will be triggered by `push` to main

Reply:
Is it possible to have wrong latest tag?

Reply:
I doubt. This job is guarded by the `if` condition that uses `release_created`.